pkg/component/worker/kubelet.go

```diff
@@ -89,13 +90,18 @@ func (k *Kubelet) Init(_ context.Context) error {
 	}
 
 	if runtime.GOOS == "linux" {
+		iptablesMode, err := iptablesutils.DetectIPTablesMode(k.K0sVars.BinDir)
+		if err != nil {
+			return fmt.Errorf("unable to detect iptables mode is being used: %w", err)
```

Comment:
I think we shouldn't return with error here, but fallback to nftables mode.

Reply:
Done